apartment.rb

```diff
@@ -0,0 +1,20 @@
+class Apartment <Tenant
```

Comment:
We probably don't want apartment inheriting from tenant. We want our models to inherit from ActiveRecord::Base.